python/rmm/rmm.py

```diff
@@ -50,8 +51,10 @@ def reinitialize(
         If True, use managed memory for device memory allocation
     initial_pool_size : int, default None
         When `pool_allocator` is True, this indicates the initial pool size in
-        bytes. None is used to indicate the default size of the underlying
-        memorypool implementation, which currently is 1/2 total GPU memory.
+        bytes. By default, 1/2 of the total GPU memory is used.
+    maximum_pool_size : int, default None
+        When `pool_allocator` is True, this indicates the maximum pool size in
```

Comment:
This is a bit pedantic, but what's the behavior when `maximum_pool_size` is set when `pool_allocator` is False? I'm assuming it's ignored?

Reply:
Yes - it is ignored

Reply:
Could we indicate such in the docstring, and possibly considering throwing a Warning or Exception if someone uses it in an ignored context?

Reply:
I made it explicit in the docstring, but I'm on the fence about throwing/warning. Could be annoying for someone wanting to quickly switch between `pool=True/False`